CHANGELOG/CHANGELOG-1.12.md

```diff
@@ -0,0 +1,73 @@
+# Changelog 1.12
+
+## 1.12.0 - 2023-05-24
+
+- The Multicast, TopologyAwareHints, NodeIPAM features are graduated from Alpha to Beta and therefore enabled by default.
```

Comment:
We should mention the new `multicast.enable` configuration parameter.

Reply:
Added.